lib/call-session.js

```diff
@@ -229,6 +229,14 @@ class CallSession extends Emitter {
           /* was a specific carrier requested */
           let voip_carrier_sid = this.req.get('X-Requested-Carrier-Sid');
           const account_sid = this.req.get('X-Account-Sid');
+          this.logger.info(`carrier requested ${voip_carrier_sid}`);
+          if (!voip_carrier_sid) {
+            if(process.env.JAMBONES_OUTDIAL_TRUNK_FAILOVER==='false') {
```

Comment:
instead of returning a 603 response when the JAMBONES_OUTDIAL_TRUNK_FAILOVER env variable is false. You might want to invert the if and follow the default behaviour if JAMBONES_OUTDIAL_TRUNK_FAILOVER is undefined or true. This way if the variable is set to false it will just skip the lookupOutboundCarrierForAccount method and continue to the next step which will return the 603 response.